fenix/app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -1206,36 +1214,31 @@ abstract class BaseBrowserFragment :
     }
 
     @VisibleForTesting
-    internal fun initializeEngineView(toolbarHeight: Int) {
+    internal fun initializeEngineView(
+        topToolbarHeight: Int,
+        bottomToolbarHeight: Int,
+    ) {
         val context = requireContext()
 
         if (!context.settings().shouldUseFixedTopToolbar && context.settings().isDynamicToolbarEnabled) {
-            getEngineView().setDynamicToolbarMaxHeight(toolbarHeight)
+            getEngineView().setDynamicToolbarMaxHeight(topToolbarHeight + bottomToolbarHeight)
 
-            val toolbarPosition = when (context.settings().toolbarPosition) {
-                ToolbarPosition.BOTTOM -> MozacToolbarPosition.BOTTOM
-                ToolbarPosition.TOP -> MozacToolbarPosition.TOP
-            }
             (getSwipeRefreshLayout().layoutParams as CoordinatorLayout.LayoutParams).behavior =
                 EngineViewClippingBehavior(
-                    context,
-                    null,
-                    getSwipeRefreshLayout(),
-                    toolbarHeight,
-                    toolbarPosition,
+                    context = context,
+                    attrs = null,
+                    engineViewParent = getSwipeRefreshLayout(),
+                    topToolbarHeight = topToolbarHeight,
+                    hasTopToolbar = !requireComponents.settings.shouldUseBottomToolbar,
```

Comment:
Just wondering if we can simply pass in the `bottomToolbarHeight` here instead of reading the pref. The check internally could be `topToobarHeight >0` ?

Reply:
good point! we already are passing the `topToobarHeight`, so could be just a local parameter